tabs/osd.js

```diff
@@ -340,10 +342,9 @@ OSD.updateDisplaySize = function() {
 OSD.msp = {
   encodeOther: function() {
     var result = [-1, OSD.data.video_system];
-    if (semver.gte(CONFIG.apiVersion, "1.21.0")) {
+    if (semver.gte(CONFIG.flightControllerVersion, "3.0.1")) {
```

Comment:
I thought we agreed before to user flight controller version? its already semver and much more visible than any other version (aside: why keep 2 versions anyway, when we release it will alway be a fc version update?)